css/team.css

```diff
@@ -225,11 +229,11 @@ h3 {
 	overflow-y: scroll;
 	height: 40%;
 }
-.modal-scroll {
+/* .modal-scroll { */
```

Comment:
Please remove commented code out if it is not needed

Reply:
done! thank you!